### ISSUE TITLE
Wavebox will not close (have to CTRL-ALT-DEL to kill)

### ISSUE THREAD

**Issue by treetexan** [wavebox/waveboxapp#628]
* Which version of Wavebox are you using? 
3.13.0

* Which Operating System are you using? 
Windows 10

Wavebox got updated recently and for the last week, refuses to close.  I have to hit control-alt-delete to close it every time.  This is maddening.  I have put in feedback on the main website and haven't heard back, so here we are.  I uninstalled all extensions and still it has this issue.

**Comment by nmat** [comment]
Can I ask you, how are you trying to close it so that we can try to replicate the issue?

I have not had problems on my 2 windows 10 machine on closing the application therefore it would be good to know how are trying to close the application.

Can you also check that the application just is not minimized to the taskbar on close?

**Comment by Thomas101** [comment]
Hi, is Wavebox still running in the tray area (bottom right)? If it is, then pressing close will only minimize the main window to the tray. Have you tried right-clicking on the tray and then using the close button to quit the app completely?
![untitled](https://user-images.githubusercontent.com/103586/37965020-83799c94-31bb-11e8-9d25-7787f17aab08.png)

Alternatively, if you want the app to quit entirely when you close the main window, disabling the tray in settings will produce this behaviour...
![screen shot 2018-03-27 at 12 34 46](https://user-images.githubusercontent.com/103586/37965054-a599328a-31bb-11e8-8ab6-15562f3eca42.png)

**Comment by luispedrofonseca** [comment]
This is a very creepy and annoying behaviour... Makes me very inclined to delete the app.

**Comment by treetexan** [comment]
Just to confirm. It was still running in the tray under an odd new icon. Problem solved. Surprised me is all.

**Comment by plcarman** [comment]
Agreed with @luispedrofonseca: pressing the close button should result in the app closing, not minimizing and running in background. This is weird to have as default behavior.

**Comment by nmat** [comment]
I think the behaviour follows the standard of how for example slack uses it.

On the other hand even if it would be default then a setting to turn it off or on or even set it the way you want would be prefered I guess.

For example slack uses close button to be set to either close or exit application. (depending on setting)

Chrome however uses close button to close the application with a warning that it will exit and close the tabs.

So its a fine line of how it should behave I guess.

**Comment by Thomas101** [comment]
We'll have a look at adding an option in that allows the app to quit whenever the main window is closed.

In the meantime to fully quit the app there are a few options...
* Disable the tray and close all open windows via the close button
* Open the tray and click the quit button
* Use the <kbd>Ctrl</kbd>+<kbd>Q</kbd> keyboard shortcut
* Enable the Window Titlebar in settings and use `File` > `Quit`
